### PR TITLE
User

### PR DESCRIPTION
**Pull request by asch8216**
Added routes for user and some front end bones. Don't commit. Let me know what you think if you have time. Do not merge yet in progress.

### CODE REVIEW

File: server/routes/user.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+router.get('/', (req, res, next) => {
+  User.findAll()
+    .then(users => res.json(users))
+    .catch((e) => {
+      console.error('error in all users route', e);
+      next(e);
+    });
+});
+
+router.get('/:id/orders', async (req, res, next) => {
```
Comment (KevinY90) [inline server/routes/user.js:17]: This looks good, I think we could exclude status: 'Cart'

File: server/routes/user.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const router = require('express').Router();
+const User = require('../database/models/user');
+const Order = require('../database/models/order');
+
+router.get('/', (req, res, next) => {
```
Comment (KevinY90) [inline server/routes/user.js]: A route to get the current user and include the orders that belong to that user

File: server/routes/user.js (hunk 1 above)
Comment (asch8216) [reply server/routes/user.js:17]: fixed this. went with the easy way but added functionality so we could create a dropdown for users to see statuses if we have time

File: client/components/User.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  orders: PropTypes.arrayOf(
+    propTypes.shape({
+      id: PropTypes.number.isRequired,
+      updatedAt: PropTypes.string,
+      createdAt: PropTypes.string,
+    }),
+  ),
+};
+
+const propTypes = {
```
Comment (KevinY90) [inline client/components/User.jsx]: Just wanted to point out that since propTypes is declared already, this will throw a syntax error. But I think you mentioned that this will be part of a different PR anyway, if so disregard!

**Review by KevinY90 — APPROVED** [review]
